docs/book/src/topics/windows.md

```diff
@@ -101,8 +101,29 @@ When creating a cluster with `Machinepool` if the Machine Pool name is longer th
 
 ### VM password and access
 The VM password is [random generated](https://cloudbase-init.readthedocs.io/en/latest/plugins.html#setting-password-main)
-by Cloudbase-init during provisioning of the VM. For Access to the VM you can use ssh which will be configured with SSH
-public key you provided during deployment.
+by Cloudbase-init during provisioning of the VM. For Access to the VM you can use ssh, which can be configured with a
+public key you provide during deployment. For example like this:
+```yaml
+apiVersion: bootstrap.cluster.x-k8s.io/v1alpha3
+kind: KubeadmConfigTemplate
+metadata:
+  name: test1-md-0
+  namespace: default
+spec:
+  template:
+    spec:
+      ...
+      users:
+      - name: username
+        groups: Administrators
+        sshAuthorizedKeys:
+        - "ssh-rsa AAAA..."
+```
+
+Refer to [SSH Access for nodes](ssh-access.md) for instructions on how to add an SSH public key for Windows instances.
+It's required to specify the SSH key using the `users` property in the Kubeadm config template. Specifying the `sshPublicKey` on `AzureMachine` / `AzureMachinePool` resources only works with Linux instances.
```

Comment:
Should this be up by line 105?

Reply:
I put it here because the sentence right before talks about how to add SSH access for windows nodes.